packages/f/font-droid-ttf/files/droid.metainfo.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Solus Developers <copyright@getsol.us> -->
+<component type="font">
+  <id>droid</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Droid</name>
+   <summary>Droid fonts. Includes Fallback and Japanese support.</summary>
```

Comment:
Minor, please make sure this lines up with the other lines.

Reply:
done